pkg/volume/rbd/rbd.go

```diff
@@ -192,6 +192,10 @@ type rbdBuilder struct {
 
 var _ volume.Builder = &rbdBuilder{}
 
+func (_ *rbdBuilder) SupportsOwnershipManagement() bool {
+	return false
```

Comment:
It is, and we have actually already tested this feature from a branch where this is set to true.